eng/tools/versioning/increment.js

```diff
@@ -70,6 +71,21 @@ async function main(argv) {
     packageJsonContents,
     newVersion
   );
+  const changelogLocation = path.join(targetPackagePath, "CHANGELOG.md");
```

Comment:
This should probably go into the VersionsUtil.js file. We want to call it for any case where we set the version not just the increment.

Reply:
Moved all those lines to a new function under version utils and calling that function from each of the scripts